ASAP-Sports/notifications/RegisterNotifications.js

```diff
@@ -0,0 +1,44 @@
+import { Permissions, Notifications } from 'expo';
+
+const PUSH_ENDPOINT = 'http://bbbf2786.ngrok.io/users/push-token';
+
```

Comment:
Do we need this file at all? Looks like garbage

Reply:
Nope gonna get rid of it I already added the necessary function in Homescreen